packages/node_modules/@webex/plugin-meetings/src/common/errors/webex-meetings-error.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @extends Error
+ * @property {number} code - Error code
+ */
+class WebexError extends Error {
```

Comment:
```suggestion
export default class WebexError extends Error {
```
Can we keep this way of exporting instead of switching back to `exports`?